app/helpers/container_project_helper/textual_summary.rb

```diff
@@ -16,6 +16,52 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_quota
+    {
+      :labels => [_("Quota Name"), _("Resource"), _("Desired"), _("Enforced"), _("Observed")],
```

Comment:
`"Quota Name"` seems redundant to me (the table itself has the "Quota" title). It could be just `"Quota"`.